README.md

```diff
@@ -17,95 +18,110 @@ This module adds a new exec provider capable of executing PowerShell commands.
 
 ##Setup
 
+###Setup Requirements
+This module requires PowerShell to be installed and the `powershell.exe` to be available in the system PATH.
+
 ###Beginning with powershell
 
-The powershell module adapts the Puppet
-[exec](http://docs.puppetlabs.com/references/stable/type.html#exec)
-resource to run PowerShell commands. To get started with the module, simply install it and declare 'powershell' in `provider` with the applicable command.
+The powershell module adapts the Puppet [exec](http://docs.puppetlabs.com/references/stable/type.html#exec) resource to run PowerShell commands. To get started, simply install the module and declare 'powershell' in `provider` with the applicable command.
 
-```
+~~~
     exec { 'RESOURCENAME':
       command   => '$(SOMECOMMAND)',
       provider  => powershell,
     }
-```
+~~~
 
 ##Usage
 
 When using `exec` resources with the `powershell` provider, the `command` parameter must be single-quoted to prevent Puppet from interpolating `$(..)`. 
 
-For instance, if you wanted to rename the Guest account
+For instance, if you wanted to rename the guest account:
 
-```
+~~~
     exec { 'rename-guest':
       command   => '$(Get-WMIObject Win32_UserAccount -Filter "Name=\'guest\'").Rename("new-guest")',
       unless    => 'if (Get-WmiObject Win32_UserAccount -Filter "Name=\'guest\'") { exit 1 }',
       provider  => powershell,
     }
-```
+~~~
 
-Note that the example uses the `unless` parameter to make the resource idempotent. The `command` is only executed if the Guest account does not exist, as indicated by
-`unless` returning 0.
+Note that the example uses the `unless` parameter to make the resource idempotent. The `command` is only executed if the guest account does not exist, as indicated by `unless` returning 0.
 
-**Note:** PowerShell variables, e.g. `$_`, must be escaped in puppet manifests either using backslashes or single quotes.
+**Note:** PowerShell variables (e.g. `$_`), must be escaped in Puppet manifests either using backslashes or single quotes.
 
 Alternatively, you can put the PowerShell code for the `command`, `onlyif`, and `unless` parameters into separate templates and then invoke the template function in the resource.
 
-```
+~~~
 exec { 'rename-guest':
   command   => template('guest/rename-guest.ps1'),
   onlyif    => template('guest/guest-exists.ps1'),
   provider  => powershell,
   logoutput => true,
 }
-```
-Each template is a PowerShell script, 
+~~~
+
+Each template is a PowerShell script.
 
-```
+~~~
 $obj = $(Get-WMIObject Win32_UserAccount -Filter "Name='Guest'")
 $obj.Rename("OtherGuest")
-```
+~~~
 
 This has the added benefit of not requiring escaping '$' in the PowerShell code.
 
 ##Reference
 
-Since the powershell module is a provider for `exec` it enables the same parameters as [`exec`](http://docs.puppetlabs.com/references/stable/type.html#exec) for PowerShell.
```

Comment:
There were previously 3 mentions of this module using exec's params in the readme, so to cut down on cruft, it was removed here. Our readme style guide also tries to avoid links as much as possible, as many of our users are reading this readme in vim or offline and markdown links make the document harder to read. The parameters that were removed from the readme were deemed not relevant to the powershell module, as previously the list of params was just copied from exec's documentation.